src/entry-dev.js

```diff
@@ -1,18 +1,22 @@
+import App from './App';
+import { NotificationsPortal } from '@redhat-cloud-services/frontend-components-notifications';
+import { Provider } from 'react-redux';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { Provider } from 'react-redux';
-import App from './App';
+import getBaseName from './Utilities/getBaseName';
```

Comment:
Can you use `getBaseName` from FEC utils package instead of custom one?

Reply:
ooo this was already in place, can do that tho (what yah see is just a rearrange)